lib/dockerize_rails/docker_commands/docker_pull.rb

```diff
@@ -0,0 +1,54 @@
+module DockerizeRails
+  module DockerCommands
+    module DockerPull
+      def self.pull_ruby
+        pull_docker_image('ruby', DRConfig.ruby_version)
+        0
+      rescue Docker::Error::NotFoundError => e
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name 'e' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->